[2025-03-01T11:20:18.047+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: customer_churn_ml_pipeline.data_storage_task manual__2025-03-01T11:18:49.240636+00:00 [queued]>
[2025-03-01T11:20:18.078+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: customer_churn_ml_pipeline.data_storage_task manual__2025-03-01T11:18:49.240636+00:00 [queued]>
[2025-03-01T11:20:18.078+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-03-01T11:20:18.114+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): data_storage_task> on 2025-03-01 11:18:49.240636+00:00
[2025-03-01T11:20:18.122+0000] {standard_task_runner.py:57} INFO - Started process 13607 to run task
[2025-03-01T11:20:18.129+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'customer_churn_ml_pipeline', 'data_storage_task', 'manual__2025-03-01T11:18:49.240636+00:00', '--job-id', '125', '--raw', '--subdir', 'DAGS_FOLDER/customer_churn.py', '--cfg-path', '/tmp/tmpgf0wdcsi']
[2025-03-01T11:20:18.135+0000] {standard_task_runner.py:85} INFO - Job 125: Subtask data_storage_task
[2025-03-01T11:20:18.181+0000] {logging_mixin.py:154} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-01T11:20:18.274+0000] {task_command.py:416} INFO - Running <TaskInstance: customer_churn_ml_pipeline.data_storage_task manual__2025-03-01T11:18:49.240636+00:00 [running]> on host 7e994a0ed489
[2025-03-01T11:20:18.535+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='customer_churn_ml_pipeline' AIRFLOW_CTX_TASK_ID='data_storage_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-01T11:18:49.240636+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-01T11:18:49.240636+00:00'
[2025-03-01T11:20:18.538+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-03-01T11:20:18.540+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', '\n\t\t\tcd /opt/***\n                        dvc pull /opt/***/data/transformed\n                        python /opt/***/executables/data_storage.py && \t\t\tgit add logs &&                        git commit -m "Updated stored data version" -a &&                         dvc push && git push\n                    ']
[2025-03-01T11:20:18.560+0000] {subprocess.py:86} INFO - Output:
[2025-03-01T11:20:23.008+0000] {subprocess.py:93} INFO - Everything is up to date.
[2025-03-01T11:20:25.089+0000] {subprocess.py:93} INFO - 2025-03-01 11:20:25,089 : INFO : Connecting to Database
[2025-03-01T11:20:25.129+0000] {subprocess.py:93} INFO - 2025-03-01 11:20:25,129 : INFO : Connecting to Database Successfull
[2025-03-01T11:20:25.130+0000] {subprocess.py:93} INFO - 2025-03-01 11:20:25,129 : INFO : Reading Initial DB Setup Script
[2025-03-01T11:20:25.130+0000] {subprocess.py:93} INFO - 2025-03-01 11:20:25,130 : INFO : Executing script to Database
[2025-03-01T11:20:25.132+0000] {subprocess.py:93} INFO - 2025-03-01 11:20:25,131 : INFO : Executed script successfully !!
[2025-03-01T11:20:25.132+0000] {subprocess.py:93} INFO - 2025-03-01 11:20:25,132 : INFO : Reading insert script template
[2025-03-01T11:20:25.133+0000] {subprocess.py:93} INFO - 2025-03-01 11:20:25,132 : INFO : Inserting data to database
[2025-03-01T11:20:30.621+0000] {subprocess.py:93} INFO - 2025-03-01 11:20:30,620 : INFO : Executed script successfully !!
[2025-03-01T11:20:30.622+0000] {subprocess.py:93} INFO - 2025-03-01 11:20:30,622 : INFO : Connection closed !!
